web/admin/src/editCell.js

```diff
@@ -84,25 +94,25 @@ export class CellEdit extends Component {
         var status = true
         if (named) {
             for (var item of this.props.sessions){
-                if (item.sessionName === named){
-                  status = false
-                  if (item.sessionName === this.props.task.sessionName){
-                      status = true
-                  }
+              if (item.sessionName.toUpperCase() === named.toUpperCase()){
+                status = false
```

Comment:
This is another place where we can replace these 4 lines with a single line: `status = (item.sessionName === this.props.task.sessionName)`

Reply:
Updated